<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>225</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>225</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ClipIt</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="central_layout">
    <item>
     <widget class="QFrame" name="capture_timer_frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="video_capture_timer_frame_layout">
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <spacer name="capture_timer_left_h_spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>113</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="capture_duration_label">
         <property name="font">
          <font>
           <pointsize>23</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="capture_timer_right_h_spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>113</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="button_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="RecordButton" name="record_button">
         <property name="text">
          <string>Start/Stop Recording</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ScreenshotButton" name="screenshot_button">
         <property name="text">
          <string>Screenshot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="OpenEditorButton" name="open_editor_button">
         <property name="text">
          <string>Open Editor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="OpenCaptureFolderButton" name="open_capture_folder_button">
         <property name="text">
          <string>Open Capture Folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SettingsButton" name="settings_button">
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScreenshotButton</class>
   <extends>QPushButton</extends>
   <header>.buttons.screenshot_button.screenshot_button</header>
  </customwidget>
  <customwidget>
   <class>RecordButton</class>
   <extends>QPushButton</extends>
   <header>.buttons.record_button.record_button</header>
  </customwidget>
  <customwidget>
   <class>SettingsButton</class>
   <extends>QPushButton</extends>
   <header>.buttons.settings_button.settings_button</header>
  </customwidget>
  <customwidget>
   <class>OpenCaptureFolderButton</class>
   <extends>QPushButton</extends>
   <header>.buttons.open_capture_folder_button.open_capture_folder_button</header>
  </customwidget>
  <customwidget>
   <class>OpenEditorButton</class>
   <extends>QPushButton</extends>
   <header>.buttons.open_editor_button.open_editor_button</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
